src/test/java/PageObject/BasePage.java

```diff
@@ -0,0 +1,31 @@
+package PageObject;
```

Comment:
Package names are written in all lower case to avoid conflict with the names of classes or interfaces.

For example `pages`